frontend/src/component/changeRequest/ChangeRequestSidebar/DraftChangeRequestActions/DraftChangeRequestActions.tsx

```diff
@@ -0,0 +1,164 @@
+    display: 'flex',
+    flexDirection: 'column',
+    '& > span:first-of-type': {
+        color: theme.palette.text.secondary,
+    },
+}));
+
+const renderOption = (
+    props: React.HTMLAttributes<HTMLLIElement>,
+    option: AvailableReviewerSchema,
+    { selected }: { selected: boolean },
+) => (
+    <StrechedLi {...props} key={option.id}>
+        <Checkbox
+            icon={<CheckBoxOutlineBlankIcon fontSize='small' />}
+            checkedIcon={<CheckBoxIcon fontSize='small' />}
+            style={{ marginRight: 8 }}
+            checked={selected}
+        />
+        <StyledOption>
+            <span>{option.name || option.username}</span>
+            <span>
+                {option.name && option.username
+                    ? option.username
+                    : option.email}
+            </span>
+        </StyledOption>
+    </StrechedLi>
+);
+
+const renderTags = (value: AvailableReviewerSchema[]) => (
+    <StyledTags>
+        {value.length > 1
+            ? `${value.length} users selected`
+            : value[0].name || value[0].username || value[0].email}
+    </StyledTags>
+);
+
+export const DraftChangeRequestActions: FC<{
+    environmentChangeRequest: ChangeRequestType;
+    reviewers: AvailableReviewerSchema[];
+    setReviewers: React.Dispatch<
+        React.SetStateAction<AvailableReviewerSchema[]>
+    >;
+    onReview: (changeState: (project: string) => Promise<void>) => void;
+    onDiscard: (id: number) => void;
+    sendToReview: (project: string) => Promise<void>;
+    disabled?: boolean;
+    setDisabled: (disabled: boolean) => void;
+}> = ({
+    environmentChangeRequest,
+    reviewers,
+    setReviewers,
+    onReview,
+    onDiscard,
+    sendToReview,
+    disabled,
+    setDisabled,
+}) => {
+    const theme = useTheme();
+    const { reviewers: availableReviewers, loading: isLoading } =
+        useAvailableChangeRequestReviewers(
+            environmentChangeRequest.project,
+            environmentChangeRequest.environment,
+        );
+
+    return (
+        <>
+            <AutocompleteVirtual
+                sx={{ ml: 'auto', width: theme.spacing(40) }}
+                size='small'
+                limitTags={3}
+                openOnFocus
+                multiple
+                disableCloseOnSelect
+                value={reviewers as AvailableReviewerSchema[]}
+                onChange={(event, newValue, reason) => {
+                    if (
+                        event.type === 'keydown' &&
+                        (event as React.KeyboardEvent).key === 'Backspace' &&
+                        reason === 'removeOption'
+                    ) {
+                        return;
+                    }
+                    setReviewers(newValue);
+                }}
+                options={availableReviewers}
+                renderOption={renderOption}
+                filterOptions={(options, { inputValue }) =>
+                    options.filter(
```

Comment:
not a big fan of scripting inside params as it leads to a lot of whitespace on the left (https://codescene.io/docs/guides/technical/complexity-trends.html#what-s-complexity-anyway)